features/schedule/backfill/feature.ts

```diff
@@ -0,0 +1,66 @@
+import { Feature, retry } from '@temporalio/harness';
+import { ScheduleClient, Connection, ScheduleOverlapPolicy } from '@temporalio/client';
+import * as assert from 'assert';
+import { randomUUID } from 'crypto';
+
+export async function workflow(arg: string): Promise<string> {
+  return arg;
```

Comment:
Is there any information provided to the workflow from the scheduler letting it know what period it's being backfilled for?
That's super useful in systems like airflow for example.
We should add that feature if it doesn't exist and test it if it does.

Reply:
I am not super familiar with airflow, what information does it expose? The only additional information a workflow has when started be a schedule is :

```
Workflow Executions started by a Schedule have the following additional properties:

* The Action's timestamp is appended to the Workflow Id.

* The TemporalScheduledStartTime [Search Attribute](https://docs.temporal.io/concepts/what-is-a-search-attribute) is added to the Workflow Execution. 

* The value is the Action's timestamp.
The TemporalScheduledById Search Attribute is added to the Workflow Execution. The value is the Schedule Id.
```

Reply:
That's great, can we test this?

Reply:
I don't think those properties listed expose any backfill information. It should definitely be tested, but does that belong here? That seems like a server test irrelevant of any sdk ?

Reply:
We're testing Temporal features as a whole here, this repo should have just been named `features`. SDKS are just a part of what makes up a feature.

Reply:
Hm OK I was under the impression we were using sdk-features for just cross sdk testing. I'll make an issue to test temporal sets the right data in workflow execution.

Reply:
https://github.com/temporalio/sdk-features/issues/202